packages/react-button/src/components/MenuButton/stories/MenuButtonAppearance.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Menu, MenuItem, MenuList, MenuPopover, MenuTrigger } from '@fluentui/react-menu';
+
+import { MenuButton } from '../../../MenuButton'; // codesandbox-dependency: @fluentui/react-button ^9.0.0-beta
+
+export const Appearance = () => (
+  <>
+    <Menu>
```

Comment:
if you would like to cut down on `Menu` code, you could simply create one `Menu` instance and reuse it. It could result in some more javascript, but less than the amoung of jsx you have in this story.

How to reuse a menu is documented already in the docs: https://6002298f95a00c00213f4d55-npdxmhsabb.chromatic.com/?path=/docs/components-menu--default#anchor-to-custom-target

Reply:
oh i gotcha. single menu instance able to be triggered by different buttons. Good to know that functionality is there. I think for this example I'm happy to keep each of them separate. If users ever want to reuse this code they will only be copying 1 out at a time.